components/Tooltip/TooltipStyles.tsx

```diff
@@ -55,6 +57,41 @@ export const Tooltip = styled.div<ToolTipStyleProps>`
 				bottom: 0;
 				transform: translate(-25%, 125%);
 			`}
+		
+		${(props) =>
+			props.preset === '1h-funding-rate' &&
+			`
+			bottom: 20px;
+			z-index: 2;
+			left: -100px;
+			transform: translate(-25%, 125%);
+		`}
+		
+		${(props) =>
+			props.preset === 'skew' &&
+			`
+			bottom: 0;
+			left: -30px;
+			transform: translate(-25%, 125%);
+			z-index: 2;
+		`}
+		
+		${(props) =>
+			props.preset === 'bottom-z-index-2-left-margin' &&
+			`
+			bottom: 0;
+			left: -60px;
+			transform: translate(-25%, 125%);
+			z-index: 2;
+		`}
+		
+		${(props) =>
+			props.preset === 'bottom-z-index-2' &&
+			`
+				bottom: 0;
+				transform: translate(-25%, 125%);
+				z-index: 2;
+			`}
```

Comment:
Instead of creating multiple presets, maybe we can handle the individual cases when using the components i.e. we can add the custom styles when we do `styled(Tooltip)`. It might clean things up.

Hardcoding it here makes the Tooltip component a little bit more complex.